<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TremoloBarDialog</class>
 <widget class="QDialog" name="TremoloBarDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tremolo Bar</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="typeComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pitchLabel">
       <property name="text">
        <string>Resultant Pitch:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="pitchComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="durationLabel">
       <property name="text">
        <string>Bend Duration:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="currentDurationLabel">
         <property name="text">
          <string>Current note duration + next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="durationSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>notes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
</ui>
